tests/event_stream_rpc_client_connection_test.c

```diff
@@ -275,19 +280,28 @@ static int s_fixture_setup(struct aws_allocator *allocator, void *ctx) {
         .type = AWS_SOCKET_STREAM,
     };
 
-    struct aws_event_stream_rpc_server_listener_options listener_options = {
-        .socket_options = &socket_options,
-        .host_name = s_test_host_name,
-        .port = s_test_port,
-        .bootstrap = test_data->server_bootstrap,
-        .user_data = test_data,
-        .on_new_connection = s_fixture_on_new_server_connection,
-        .on_connection_shutdown = s_fixture_on_server_connection_shutdown,
-        .on_destroy_callback = s_on_listener_destroy,
-    };
-
-    test_data->listener = aws_event_stream_rpc_server_new_listener(allocator, &listener_options);
-    ASSERT_NOT_NULL(test_data->listener);
+    /* Find a random open port */
+    uint16_t test_port = 0;
```

Comment:
let's not do this. Can you use a unix domain socket instead? Open ports are hard to predict, and you don't have to deal with it for unix domain sockets.

Reply:
I was just trying to fix the tests.
I'd assumed you were using IP sockets for a reason.
I'll switch to domain sockets like our other tests use

Reply:
nevermind. not doing this right now, need to forge ahead
these testing changes had no relation to my fixes anyway
with domain sockets tests are freezing in host-resolver shutdown and I don't have time to debug :|